### ISSUE TITLE
Create a sheet for units of troops for the warfare mechanic.

### ISSUE THREAD

**Issue by avorial** [3darkman/foundryvtt-chroniclesystem#23]
Ideally

A command button that was linked to the general of that army that gave it a warfare command. As all units have a modifier based on how trained they are. 

Traits - As all have 3 traits and a few changes can give them one more. 

Special Traits - Such as on horseback

Damage, they would cause.

Extra Items they would have to modify other rolls or effects.

**Comment by 3darkman** [comment]
hello guys,

I was thinking of implementing this for this 0.5.0 version, but I'll leave it for another version, to know exactly what they differ from the standard characters, because it seems to me that the character sheet suits them well, maybe just making the appearance more simplified. I wanted to hear more about that from you.

**Comment by jesuisfrog** [comment]
The option to have de-cluttered sheets would be great, not just for units but even for NPCs and PCs. Maybe we could have a set of checkboxes on the attribute page/tab (Since that would be the one page/tab you'd always want visible) that would hide any unchecked pages/tabs. 

This would allow for more streamlined sheets without having to create new sheets.

**Comment by avorial** [comment]
Can you drop character sheets on character sheets. 
It would be great if we could drop them on the house sheet, and then just click to open them.

They also need to clearly have a (Type/experince) at the top as those are the key rolls needed when they are used in warfare.

**Comment by 3darkman** [comment]
yeah... I really need to read the part about warfare calmly again, I don't remember how the units are anymore... 

regarding adding them to the house sheet, that's fine, the functionality is already done with the members, just add a new type, maybe for a new tab.

**Comment by 3darkman** [comment]
preview: a new type of item, which will be added to unit, Unit Type.
![image](https://user-images.githubusercontent.com/20743586/203876480-352b398e-fbfb-4069-b7be-7f4f26b12945.png)

**Comment by 3darkman** [comment]
a new preview. note that this is not the final layout, basically these are tests to verify that the fields are behaving correctly. 

P.s.: available abilities are added automatically by linked unit types.

![image](https://user-images.githubusercontent.com/20743586/205450693-da85f0cf-4729-48f0-abb9-504bf8b58ae2.png)

**Comment by avorial** [comment]
I’m fine to drag them all as there are traits that would let you add one or special units can have any Also if we could have a modifier for weapons and or dogs ect. On Dec 3, 2022, at 8:18 AM, Kirlian Silvestre ***@***.***> wrote:﻿
a new preview. note that this is not the final layout, basically these are tests to verify that the fields are behaving correctly.
P.s.: available abilities are added automatically by linked unit types.

**Comment by leandropug** [comment]
Hello [3darkman](https://github.com/3darkman) , there is a way to use and test those itens of Units? I never found them on my version ( im on Foundry 11 )

**Comment by avorial** [comment]
As we are coming on a year of no pushed updates, I find it is easier to talk to him on discord.

**Comment by 3darkman** [comment]
is done, will be available in the next version.

<img width="898" height="595" alt="Image" src="https://github.com/user-attachments/assets/4ad1c683-9d78-4666-9a03-0778b39c495b" />
